tests/Custom/TimeoutTest.php

```diff
@@ -48,12 +51,18 @@ public function testShortTimeoutDoesNotWaitForElementToAppear(): void
 
     public function testLongTimeoutWaitsForElementToAppear(): void
     {
-        $this->getSession()->getDriver()->setTimeouts(['implicit' => 5000]);
+        $this->getDriver()->setTimeouts(['implicit' => 5000]);
 
         $this->getSession()->visit($this->pathTo('/js_test.html'));
         $this->findById('waitable')->click();
         $element = $this->getSession()->getPage()->find('css', '#waitable > div');
 
         $this->assertNotNull($element);
     }
+
+    private function getDriver(): WebdriverClassicDriver
+    {
+        /** @phpstan-ignore-next-line */
```

Comment:
you should instead add an assert, as I did in Selenium2Driver. This is better than ignoring errors (in development mode, it will actually fail if the invariant is wrong)

Reply:
I mean as I did in https://github.com/minkphp/MinkSelenium2Driver/pull/369 which is not merged yet.

Reply:
I'm not sure I see the difference - in both cases an error is triggered (unless assertions have been disabled), and afaik for assertions the failure message won't be very descriptive, unless I'm missing something?

Reply:
the difference is that `@phpstan-ignore-next-line` ignores *any* error happening on a next line.
An assertion will then make phpstan aware that we deal with a specific kind of value and it will continue its analysis based on that, and will still report other errors if we do them.
and this is not really a case where we expect the assertion to fail (due to the way we setup the testsuite, we know which driver we have)